config/services.yml

```diff
@@ -54,7 +54,11 @@ services:
 
   # AcquiaCloudApi services.
   AcquiaCloudApi\Connector\Client: ~
-  AcquiaCloudApi\Connector\ConnectorInterface: ~
+  AcquiaCloudApi\Connector\Connector:
+    arguments:
+      $config: ['@=service("datastore.cloud").get("key")', '@=service("datastore.cloud").get("secret")']
```

Comment:
I'm pretty sure that splitting this into an interface (ConnectorInterface) and subclass (Connector) defined as an alias is the right move, since it follows the pattern further down for Symfony Input and Output interfaces.

Manually defining the argument is necessary to fix [this test failure](https://travis-ci.com/github/acquia/cli/jobs/456247653#L3099). However: I'm pretty sure this is not working like we expect. Tests pass and everything works fine, but in my testing I could define _any_ value as this argument and it would still work.

Possibly this is because of how we use a factory method to produce a Client service, and that needs to be refactored too? I hate Symfony wiring 😬

Reply:
I think we should merge this as-is and we'll clean up the wiring in a follow-up issue: https://github.com/acquia/cli/pull/380